AmazonChimeSDK/AmazonChimeSDK/internal/utils/Converters.swift

```diff
@@ -36,7 +40,7 @@ import Foundation
                 return .meetingFailed
             case .meetingEnded:
                 return .meetingEnded
-            default:
+            case .unknown:
```

Comment:
I guess this is okay, but shouldn't use `default` here a bit more safe?

Reply:
Using `default` actually makes it unsafe, when adding a new `EventName` case, we could miss adding the conversion here. We should always keep the switch exhausted.